tiledb/sm/array_schema/array_schema.h

```diff
@@ -589,6 +586,14 @@ class ArraySchema {
       std::optional<std::pair<uint64_t, uint64_t>> timestamp_range =
           std::nullopt);
 
+  /** Returns the enumeration map. */
+  const tdb::pmr::unordered_map<std::string, shared_ptr<const Enumeration>>&
+  enumeration_map() const;
+
+  /** Returns the dimension labels. */
+  const tdb::pmr::vector<shared_ptr<const DimensionLabel>>& dimension_labels()
+      const;
```

Comment:
These functions should probably be `inline`, as they're simple accessors. Our old code did not do this very often; we've been shifting it.